app/components/alert_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= tag.div(
+      **tag_options,
+      class: ['usa-alert', "usa-alert--#{type}", *tag_options[:class]],
```

Comment:
do we want to a method for like `alert_type_class` so we could enumerate the types for easier grepping?

Reply:
> do we want to a method for like `alert_type_class` so we could enumerate the types for easier grepping?

Yeah, I could do that 👍

Reply:
> do we want to a method for like `alert_type_class` so we could enumerate the types for easier grepping?

Updated in 3fbb923. Thinking I might also make a method for the full class name, since the logic is non-trivial.

Reply:
>Thinking I might also make a method for the full class name, since the logic is non-trivial.

Done in 3727e07.